ingestion/src/metadata/ingestion/source/database/common_db_source.py

```diff
@@ -129,11 +129,16 @@ def get_database_names(self) -> Iterable[str]:
         Sources with multiple databases should overwrite this and
         apply the necessary filters.
         """
-        custom_database_name = self.service_connection.__dict__.get("databaseName")
-
-        database_name = self.service_connection.__dict__.get(
-            "database", custom_database_name or "default"
+        custom_database_name = self.service_connection.__dict__.get(
+            "databaseName", None
         )
+
+        database_name = self.service_connection.__dict__.get("database", "default")
```

Comment:
then what is the difference between "database" and "databaseName" 🤔 IMO both are serving the same purpose if you want we can rename the `database` field to `databaseName`